### PR TITLE
Feature/api

### LINKED ISSUE #42: Send post request to API

### PR DESCRIPTION
**Pull request by dlouise64**
# What has changed and why?

- Removed a few values in CSP (public/index.html) which were not being used
- Removed HashRouter as no longer using redux wizard on forms, now the last page is being accessed via window.location

# How to test this change

- [ ] has automated tests
- [x] at least one a reviewer ran the code
- [x] tested in small screen
- [x] tested in Chrome
- [x] tested in Firefox
- [ ] tested in IE
- [ ] tested in Safari

closes #42

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ export class App extends Component {
           <h1>The Rates Rebate Scheme provides a discount to low-income<br/> homeowners on the cost of their rates.</h1>
           <p>If you are a low-income homeowner you could get a discount or partial refund of up to $620 on your property rates.</p>
 
-          <Link className="btn btn-primary" to={'apply'}>Apply now</Link>
+          <Link className="btn btn-primary" to='/apply'>Apply now</Link>
```
Comment (Br3nda) [inline src/components/App.js]: will this link work on github.io ? It won't be on /

Comment (dlouise64) [reply src/components/App.js]: @Br3nda Oh yeah! Thanks for that.

**Comment by Br3nda** [comment]
noting I'm still getting the address loook up not working -- but we can look at that when we map it to valuation_id.

**Comment by Br3nda** [comment]
I think it's the internet being crap here -- but hey the app needs to work when the internet is snails pace and flaky

**Comment by Br3nda** [comment]
Sweet. all posted and came through on the rails app
image

File: PULL_REQUEST_TEMPLATE [hunk 2]
```diff
@@ -5,5 +5,9 @@
 - [ ] has automated tests
 - [ ] at least one a reviewer ran the code
 - [ ] tested in small screen
+- [ ] tested in Chrome
+- [ ] tested in Firefox
+- [ ] tested in IE
+- [ ] tested in Safari
```
Comment (Br3nda) [inline PULL_REQUEST_TEMPLATE]: :sparkles: 
Let's copy this to the boilerplate.

**Pull request merged**